deployment/eclipse-kuksa/build_kuksa_appstore.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+######################################################################
+# Copyright (c) 2019 Bosch Software Innovations GmbH [and others]
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+########################################################################
+
+# Make the script fail if a command fails
+set -e
+
+SCRIPTPATH=$(dirname "$(readlink -f "$0")")
+
+DOCKER_REGISTRY_SECRET=$1
+DOCKER_REGISTRY_SERVER=$2
+DOCKER_REGISTRY_USERNAME=$3
+DOCKER_REGISTRY_PASSWORD=$4
+DOCKER_REGISTRY_EMAIL=$5
+
+APPSTORE_PATH=$SCRIPTPATH/kuksa.cloud/kuksa-appstore
+DOCKER_IMAGE_NAME="kuksa-appstore"
+
+. $SCRIPTPATH/../kubernetes/includes/allIncludes.inc
+
+echo
+echo "##############################################################"
+echo "##############################################################"
+echo "############### Eclipse Kuksa Cloud deployment ###############"
+echo "##############################################################"
+echo "##############################################################"
+
+echo
+echo "########## Initialization ##########"
+
+cd $SCRIPTPATH
+
+echo 
+echo "##### Check out repository #####"
+git clone https://github.com/eclipse/kuksa.cloud.git
```

Comment:
May we assume that the user already checked out the repository to access this file (build_kuksa_appstore.sh)? In that case we could use the path to the app-store without downloading the whole repository again for every build of the app-store.